autogluon/utils/tabular/ml/models/lr/lr_model.py

```diff
@@ -38,10 +38,12 @@ def _get_types_of_features(self, df):
         """
         if self.types_of_features is not None:
             logger.warning("Attempting to _get_types_of_features for LRModel, but previously already did this.")
-        categorical_featnames = self.__get_feature_type_if_present('object') + self.__get_feature_type_if_present('bool')
-        continuous_featnames = self.__get_feature_type_if_present('float') + self.__get_feature_type_if_present('int') + self.__get_feature_type_if_present(
-            'datetime')
-        language_featnames = self.feature_types_metadata['nlp']
+
+        feature_types = self.feature_types_metadata.feature_types_raw
+
+        categorical_featnames = feature_types['category'] + feature_types['object'] + feature_types['bool']
+        continuous_featnames = feature_types['float'] + feature_types['int']  # + self.__get_feature_type_if_present('datetime')
+        language_featnames = []  # TODO: Disabled currently, have to pass raw text data features here to function properly
```

Comment:
Should we merge this PR before this is fixed
Seems like a quality-of-life PR shouldn't be breaking functionality, or is there some urgency to merge this?

Reply:
As explained offline, this is not breaking.